addons/hr_attendance/static/src/scss/views/hr_attendance.scss

```diff
@@ -11,3 +11,14 @@
 .o_nocontent_help:has(.oe_view_nocontent_attendance_mobile) {
     position: relative;
 }
+
+.o_workplace_map_container {
+    width: 100%;
+    height: 600px;
+}
+
+#o_workplace_map {
+    width: 100%;
+    height: 100%;
+    border-radius: 8px;
+}
```

Comment:
Generally, we start class names with o_. You can either use that class or rename it. Also, check on small and  devices